debian/control

```diff
@@ -0,0 +1,36 @@
+Source: python-securesystemslib
+Section: devel
+Priority: optional
+Maintainer: NYU Secure Systems Lab <securesystemslab@googlegroups.com>
+Uploaders:
+ Santiago Torres-Arias <santiago@nyu.edu>,
+ Holger Levsen <holger@debian.org>,
+ Lukas Puehringer <lukpueh@nyu.edu>,
```

Comment:
You should verify Lukas's email address.  I thought his NYU email was `lukas.puehringer@nyu.edu`.

Reply:
Noted. will fix.